docs/experience/webvitals.md

```diff
@@ -0,0 +1,55 @@
+title: Web Vitals
+description: Gathering information about web vitals - essential site health metrics
+
+Supported by Chrome, the [Web Vitals](https://web.dev/vitals/) is a Google initiative to provide unified guidance for metrics that are crucial for delivering the best user experience for the web-based applications. 
```

Comment:
>for the web-based application

I would suggest "for websites and webapps". I think we used those terms in other docs pages.

Reply:
Done.